src/c/profile/transport/tcp/tcp_transport_linux.c

```diff
@@ -14,6 +20,8 @@ bool uxr_init_tcp_platform(struct uxrTCPPlatform* platform, const char* ip, uint
     platform->poll_fd.fd = socket(PF_INET, SOCK_STREAM, 0);
     if (-1 != platform->poll_fd.fd)
     {
+        signal(SIGPIPE, sigpipe_handler);
```

Comment:
The current implementation of `sigpipe_handler` is empty. Is this call really necessary?

Reply:
Yes, it is necessary. SIGPIPE signal could terminate the process if we do not handle it, and the simplest way is to use an empty handler.